docker/Dockerfile.Server

```diff
@@ -2,7 +2,7 @@
 # From the parent folder enter:
 # docker image build -t mailman:server -f ./Dockerfile.Server .
 
-FROM microsoft/dotnet:2.1-sdk AS builder
+FROM microsoft/dotnet:2.2-sdk AS builder
```

Comment:
Should this be 3.0?

Reply:
yes, you are right.  
Updated now - this affects 4 files.